packages/jest-snapshot/src/types.ts

```diff
@@ -47,6 +54,21 @@ export interface SnapshotMatchers<R extends void | Promise<void>, T> {
     propertyMatchers: Partial<U>,
     hint?: string,
   ): R;
+  /**
+   * This ensures that a value matches the specific snapshot.
+   * Instead of use current test name in global state, it will use the specific name to find the snapshot.
+   * Check out [the Snapshot Testing guide](https://jestjs.io/docs/snapshot-testing) for more information.
```

Comment:
The guide does not mention `. toMatchNamedSnapshot()`, or?

```suggestion
```

Reply:
I think I will remove the comment. There is no detail to explain more about `toMatchNamedSnapshot`.